internal/auth/auth.go

```diff
@@ -33,12 +33,15 @@ import (
 )
 
 var (
-	ErrUserNotFound = errors.New("user not found")
+	ErrUserNotFound     = errors.New("user not found")
+	ErrUsernameConflict = errors.New("this user is defined multiple times, please talk to the operator")
```

Comment:
I would just tell what happened and not what to do:

```suggestion
	ErrUsernameConflict = errors.New("username defined multiple times")
```

Reply:
ok